doc/source/skips/4-transition-to-v2.rst

```diff
@@ -143,29 +150,60 @@ namespace with a version number on it, such as OpenCV (imported as ``cv2``) and
 BeautifulSoup (imported as ``bs4``), Jinja (``jinja2``) and psycopg (currently
 imported as ``psycopg2``). Further afield, R's ggplot is used as ``ggplot2``.
 
+.. _skip4_implementation:
+
 Implementation
 --------------
 
-The details of the proposal are as follows:
-
-- scikit-image 0.19 will be followed by scikit-image 1.0. Every deprecation
-  message will be removed from 1.0, and the API will be considered the
-  scikit-image 1.0 API.
-- After 1.0, the main branch will be changed to (a) change the import name to
-  skimage2, (b) change the package name to skimage2, and (c) change the version
-  number to 2.0-dev.
-- There will be *no* "scikit-image" package on PyPI with version 2.0. Users who
-  ``pip install scikit-image`` will always get the 1.x version of the package.
-  To install scikit-image 2.0, users will need to ``pip install skimage2``,
-  ``conda install skimage2``, or similar.
-- After consensus has been reached on the new API, skimage2 will be released.
-- Following the release of skimage2, a release of scikit-image 1.1 is made. This
-  release is identical to 1.0 (including bugfixes) but will advise users to
-  either (a) upgrade to skimage2 or (b) pin the package to ``scikit-image<1.1``
-  to avoid the warning.
-- scikit-image 1.0.x and 1.1.x will receive critical bug fixes for an
-  unspecified period of time, depending on the severity of the bug and the
-  amount of effort involved.
+As a first execution step of this SKIP, scikit-image 1.0 will be released, celebrating the the maturity of the project.
+
+First phase: Building `skimage2`
+................................
+
+Afterwards, a new empty ``skimage2`` namespace will be created in our repository alongside our ``skimage`` namespace.
+It will be marked as experimental – importing it will warn that content in ``skimage2`` is still unstable.
+Initially, this namespace will not be included full releases (versioned 1.x) on PyPI and elsewhere, but may already be included in our nightly releases.
+Towards the end of this phase, ``skimage2`` should be included in full releases to help downstream libraries and users try it out.
+
+With the new namespace available, we will start building the new API inside it.
+It will – where possible – wrap the existing implementation in the existing ``skimage`` namespace but will have its own independent test suite.
```

Comment:
```suggestion
It will – where possible – wrap the implementation existing in the ``skimage`` namespace but will have its own independent test suite.
```
Alternatively: "wrap the existing/old implementation (i.e., that in the ``skimage`` namespace)"